### ISSUE TITLE
Can't create test if there's a space in the table name

### ISSUE THREAD

**Issue by sfirke** [open-metadata/OpenMetadata#8482]
**Affected module**
Backend?

**Describe the bug**
I'm trying to create a test for a table called `Fire Incidents`.  It fails.  Looking at logs it seems that sometimes the space is being turned into `+` and sometimes into `%20%`.  Compare the first line, which has both versions, and the last line:
```
10.216.7.4 - - [01/Nov/2022:16:23:15 +0000] "GET /api/v1/testCase?fields=testDefinition&limit=100000&entityLink=%3C%23E::table::<service>.<database>.dbo.Fire+Incidents%3E HTTP/1.1" 200 32 "https://<my-domain>/data-quality-test/table/<service>.<database>.dbo.Fire%20Incidents" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:102.0) Gecko/20100101 Firefox/102.0" 45
172.16.240.4 - - [01/Nov/2022:16:23:19 +0000] "GET /api/v1/version HTTP/1.1" 200 108 "-" "python-requests/2.28.0" 0
172.16.240.4 - - [01/Nov/2022:16:23:19 +0000] "GET /api/v1/services/databaseServices/name/<service> HTTP/1.1" 200 1084 "-" "python-requests/2.28.0" 31
172.16.240.4 - - [01/Nov/2022:16:23:20 +0000] "GET /api/v1/version HTTP/1.1" 200 108 "-" "python-requests/2.28.0" 0
172.16.240.4 - - [01/Nov/2022:16:23:20 +0000] "GET /api/v1/testSuite/name/AirQualitySensors HTTP/1.1" 200 355 "-" "python-requests/2.28.0" 30
172.16.240.4 - - [01/Nov/2022:16:23:21 +0000] "GET /api/v1/version HTTP/1.1" 200 108 "-" "python-requests/2.28.0" 1
10.216.7.4 - - [01/Nov/2022:16:23:21 +0000] "POST /api/v1/testCase HTTP/1.1" 404 90 "https://<my-domain>/data-quality-test/table/<service>.<database>.dbo.Fire%20Incidents" "Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:102.0) Gecko/20100101 Firefox/102.0" 32
```

In the GUI I get a popup like this when I try to save the test: 
![image](https://user-images.githubusercontent.com/7569808/199286167-533bd993-3c0c-4543-8c04-67d458ba9909.png)

**To Reproduce**

Create a test suite (succeeds) and test (fails) for a table with a space in its name.

**Version:**
 - OpenMetadata version: 0.12.2

**Additional context**
I'm ingesting MSSQL if that matters.

**Comment by sfirke** [comment]
Possibly related to https://github.com/open-metadata/OpenMetadata/issues/7623 although that dealt with column names not table names.

**Comment by harshach** [comment]
@Sachin-chaurasiya @TeddyCr ^

**Comment by TeddyCr** [comment]
This should be fixed in 0.13.2. @sfirke let's know if you are still experiencing this issue.
